assets/base/config.json

```diff
@@ -13,9 +13,9 @@
     "MobileNoticeURL": "https://about.mattermost.com/mobile-notice-txt/",
     "RudderApiKey": "",
 
-    "SentryEnabled": false,
-    "SentryDsnIos": "",
-    "SentryDsnAndroid": "",
+    "SentryEnabled": true,
+    "SentryDsnIos": "https://8166acfeeced43de98bc11f217d871c5@o1347733.ingest.sentry.io/6734065",
+    "SentryDsnAndroid": "https://8166acfeeced43de98bc11f217d871c5@o1347733.ingest.sentry.io/6734065",
```

Comment:
This should not be modified, see the dev docs on how to override this values in your local config

Reply:
By the dev docs, is it [this one](https://docs.mattermost.com/configure/configuration-settings.html) ? For the mobile app, did we take a subset of the server config and added it in our code base?